rust/migrate-wicked/src/wireless.rs

```diff
@@ -73,6 +82,289 @@ pub struct Wep {
     pub key: Vec<String>,
 }
 
+#[derive(Debug, PartialEq, Serialize, Deserialize)]
+pub struct WpaEap {
+    pub method: WickedEapMethods,
+    pub identity: Option<String>,
+    pub phase1: Option<Phase1>,
+    pub phase2: Option<Phase2>,
+    pub anonid: Option<String>,
+    pub tls: Option<WickedTLS>,
```

Comment:
```suggestion
    pub tls: Option<EapTLS>,
```
I mean, otherwise we should also call `EapAuthProto` => `WickedEapAuthProto` and so on, right?